tests/Platforms/OraclePlatformTest.php

```diff
@@ -823,6 +824,72 @@ public function testQuotedTableNames(): void
         self::assertEquals($createTriggerStatement, $sql[3]);
     }
 
+    public function testDropIndexWithoutQuotedIndexName(): void
+    {
+        $sql = $this->platform->getDropIndexSQL('IDX_NAME');
+        self::assertEquals('DROP INDEX IDX_NAME', $sql);
```

Comment:
Please do not add unit tests for SQL as they are often wrong. Add integration tests instead. See more details in the [guidelines](https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/testing.html#integration-tests).